corpus/templates/pages/team.html

```diff
@@ -0,0 +1,399 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block content %}
+  <head>
+    <style>
+      .flex-container {
+        display: flex;
+        flex-flow: row wrap;
+        align-items: flex-start;
+        align-content: space-between;
+        align-content: center;
+        text-align: center;
+      }
+
+      .profile-pic {
+        height: 150px;
+        width: 170px;
+        border-radius: 50%;
+      }
+
+      .title {
+        font-family: 'Ubuntu';
+        font-style: normal;
+        font-weight: 400;
+        margin-top: 20px;
+        margin-bottom: 20px;
+        text-align: center;
+        font-size: 50px;
+      }
+
+      .section-heading {
+        margin-top: 20px;
+        font-family: 'Ubuntu';
+        font-style: normal;
+        text-align: center;
+        font-size: 30px;
+      }
+
+      .margins {
+        margin-top: 15px;
+        margin-left: 20px;
+        margin-right: 20px;
+        margin-bottom: 15px;
+      }
+
+      .card-post {
+        font-family: 'Ubuntu';
+        font-style: normal;
+        font-weight: normal;
+        margin-top: 15px;
+        margin-bottom: 5px;
+        text-align: center;
+        font-size: 25px;
+      }
+
+      .card-title {
+        font-family: 'Ubuntu';
+        font-style: normal;
+        font-weight: bold;
+        margin-top: 15px;
+        margin-bottom: 5px;
+        text-align: center;
+        font-size: 25px;
+      }
+    </style>
+
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.12.1/css/all.css" crossorigin="anonymous">
```

Comment:
What is FontAwesome being used for here? Could you maybe put in a screenshot?

Reply:
It's being used for the Github and Linkedin icons in the buttons

![image](https://github.com/IEEE-NITK/corpus/assets/113475659/41ab8037-1fc2-420c-804f-81b26413246f)

Reply:
Can we just use the SVG files for both of these instead of importing the entire FA library? Will save some network overhead.